pyproject.toml

```diff
@@ -1,52 +1,141 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+  # TODO: keep in sync with [project.dependencies]
+  "wheel>=0.43",
+  "build>=1.1",
+  "setuptools>=63.0.0",
+  "scikit-build>=0.17.0",
+  "ninja>=1.11.1; platform_system!='Windows'",
```

Comment:
We used `ninja` on Windows before, what is the reason for moving away from `ninja`? What build system replaces `ninja` on Windows?

Reply:
On Windows Ninja is distributed with MSVC